sympy/geometry/ellipse.py

```diff
@@ -1339,6 +1341,13 @@ class Circle(Ellipse):
     >>> c2.hradius, c2.vradius, c2.radius, c2.center
     (sqrt(2)/2, sqrt(2)/2, sqrt(2)/2, Point2D(1/2, 1/2))
 
+    >>> # a circle object is returned
+    >>> from sympy.abc import x, y, a, b
+    >>> Circle(equation = x ** 2 + y ** 2 - 25)
+    Circle(Point2D(0, 0), 5)
+    >>> Circle(equation = a ** 2 + b ** 2 - 25, x='a', y='b')
```

Comment:
Can you modify the code in a way such that user doesn't have to specify `equation=....` everytime ?
The first argument to `Circle` should be interpreted as the equation of circle.

Reply:
@ArifAhmed1995  The arguments `equation`, `x` and `y` are being passed in `**kwargs` here i.e. a dictionary. This is why the user has to specify `equation = ...` in the beginning. 
Note : I tried to pass them initially as `*args` but it eventually led to a lot of errors.
I think it'll be best to make use of  `**kwargs` and not `*args` (as done already).

What do you think?

Reply:
@ArifAhmed1995  I had thought of that initially, also because that would've made the code a bit simpler too. Turns out it will have more disadvantages :-
1. Everywhere, we will have to change the format of the usage of `Circle`.(wherever `Circle` is used, we would have to add an equation for it)
2. If we don't want to do `1.` then how will python understand whether to use the incoming argument as `equation` or as `*args`?
One way to resolve that would be to specify `equation = ...` in the beginning when we want  `equation` and not specifying anything when we want `*args`. 
But this is where we had started. We already have this in `**kwargs`. 
This is why I think making `equation` compulsory is not a good idea.

I apologise if I misunderstood your suggestion(Please confirm about that).
What do you think?

Reply:
How about having a compulsory first argument and then checking if it is an instance of `Point` class or not? Depending on that you either set an equation or you set a center point.

Reply:
@ArifAhmed1995  Done now. Please take a look.

Reply:
@ArifAhmed1995  Sorry for the delay. I had a few tests going on in college. I've corrected the failing tests. They all pass now (at least in my local system).
>I don't think you are running tests locally before pushing commits. 

You are right. I couldn't test the last commit (f74ac8e) on my local system although I should've. Will not do this again. Thanks for the input.

Please take a look now.

Reply:
@avishrivastava11 Seems fine to me now.